src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -674,3 +674,14 @@ def create(self, vol_name: str, subvol_name: str, **kwargs):
             )
 
         return f'Subvolume {subvol_name} created successfully'
+
+    def set(self, vol_name: str, subvol_name: str, size: str):
+        if size:
```

Comment:
size is a string?

Reply:
they have `infinite` as a size value when quota is not set.